src/borg/archiver/create_cmd.py

```diff
@@ -192,6 +194,8 @@ def create_inner(archive, cache, fso):
                     archive.save(comment=args.comment, timestamp=args.timestamp, stats=archive.stats)
                     args.stats |= args.json
                     if args.stats:
+                        archive.stats.chunking_time = format_timedelta(timedelta(seconds=archive.stats.chunking_time))
+                        archive.stats.hashing_time = format_timedelta(timedelta(seconds=archive.stats.hashing_time))
                         if args.json:
                             json_print(basic_json_data(manifest, cache=cache, extra={"archive": archive}))
```

Comment:
maybe looks how basic_json_data() works and what str(archive.stats) invokes.

for json, check if and how other values are formatted.

for str, check how other values are formatted and maybe do it there in the same way.